source/backend/dal/Repositories/AcquisitionFileRepository.cs

```diff
@@ -911,6 +911,13 @@ private IQueryable<PimsAcquisitionFile> GetCommonAcquisitionFileQueryDeep(Acquis
                 || acq.PimsCompensationRequisitions.Any(cr => EF.Functions.Like(cr.AlternateProject.Code, $"%{filter.ProjectNameOrNumber}%") || EF.Functions.Like(cr.AlternateProject.Description, $"%{filter.ProjectNameOrNumber}%")));
             }
 
+            if (!string.IsNullOrWhiteSpace(filter.OwnerName))
+            {
+                predicate = predicate.And(acq => acq.PimsAcquisitionOwners.Any(ownr => EF.Functions.Like(ownr.GivenName, $"%{filter.OwnerName}%")
+                                    || EF.Functions.Like(ownr.LastNameAndCorpName, $"%{filter.OwnerName}%")
+                                    || EF.Functions.Like(ownr.OtherName, $"%{filter.OwnerName}%")));
+            }
+
```

Comment:
@eddherrera - the following AC in the story doesn't seem to be met:
> “Owners” data (owner solicitor, owner representative) becomes a searchable parameter (see screenshot)

This only searches for owners (ignoring the owner solicitor and representative fields). Did you get confirmation from Jordan that this would be as expected? I did not see any comments in the ticket. 

If so please update the ACs to reflect what you and Jordan agreed